### PR TITLE
gh-99387: Update `pathlib.Path.absolute()` docs to add `versionadded`

### PR DESCRIPTION
**Pull request by Reddragonemperor58**
#99387: committed changes to tech documentation under the path.absolute() function

### CODE REVIEW

File: Doc/library/pathlib.rst [hunk 1]
```diff
@@ -1217,6 +1217,8 @@ call fails (for example because the path doesn't exist).
       >>> p.absolute()
       PosixPath('/home/antoine/pathlib/tests')
 
+   .. versionadded:: 3.11
```
Comment (AA-Turner) [inline Doc/library/pathlib.rst:1220]:
```suggestion
   .. versionadded:: 3.4
```
But per Brandt, this likely isn't needed at all.

A

**Comment by Reddragonemperor58** [comment]
**could someone help me out!!**
I am new to **open source** 
and **### I tried my first pull request today and couldn't understand where I was going wrong.**


Screenshot (455)

File: Doc/library/pathlib.rst (hunk 1 above)
Comment (Reddragonemperor58) [reply Doc/library/pathlib.rst:1220]: yeah my bad I was committing changes when he posted and I didn't see that but still
@AA-Turner isn't it 3.11?

Comment (AA-Turner) [reply Doc/library/pathlib.rst:1220]: This is marked as resolved, though you haven't pushed the change yet -- is this intentional?

A

Comment (Reddragonemperor58) [reply Doc/library/pathlib.rst:1220]: isn't it version 3.11?

Comment (Reddragonemperor58) [reply Doc/library/pathlib.rst:1220]: No i didn't understand the term!
i just clicked on it and was waiting for your reply then I saw it was changed to unresolved;😅
and the resolved thing was trailing whitespaces

**Comment by tudortimi** [comment]
@AA-Turner IMO, kind of strange to mark it as "added in 3.4", since pathlib itself was added in 3.4. The philosophical issue in #99387 remains.

**Comment by brettcannon** [comment]
The decision in the issue is to not make any changes. Thanks anyway for the PR, @Reddragonemperor58 !

**Pull request closed without merging**